Build - gradlew checkstyle failed on windows

On `develop`, if I run `gradlew checkstyle`, it failed.
Platform: windows

```
$ gradlew checkstyle --stacktrace                                                                   > Task :checkstyle FAILED                                                                                                                                                                               FAILURE: Build failed with an exception.                                                                                                                                                                * What went wrong:
Execution failed for task ':checkstyle'.
> Failed to create MD5 hash for file content.                                                       
* Try:
Run with --info or --debug option to get more log output. Run with --scan to get full insights.     
* Exception is:
org.gradle.api.tasks.TaskExecutionException: Execution failed for task ':checkstyle'.
        at org.gradle.api.internal.tasks.execution.CatchExceptionTaskExecuter.execute(CatchExceptionTaskExecuter.java:38)
        at org.gradle.api.internal.tasks.execution.EventFiringTaskExecuter$1.executeTask(EventFiringTaskExecuter.java:73)
...
        ... 24 more
Caused by: java.io.IOException: The process cannot access the file because another process has locked a portion of the file
        at org.gradle.internal.hash.DefaultStreamHasher.doHash(DefaultStreamHasher.java:52)
        at org.gradle.internal.hash.DefaultStreamHasher.hash(DefaultStreamHasher.java:35)
        ... 58 more
```

--- a/gradle/wrapper/gradle-wrapper.properties
+++ b/gradle/wrapper/gradle-wrapper.properties
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-5.4.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-5.6.3-bin.zip
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists

